peps/pep-0725.rst

```diff
@@ -363,19 +411,14 @@ SciPy 1.10:
       "virtual:compiler/cpp",
       "virtual:compiler/fortran",
       "pkg:generic/ninja",
+      "pkg:generic/pkg-config",
     ]
     host-requires = [
       "virtual:interface/blas",
       "virtual:interface/lapack",  # >=3.7.1 (can't express version ranges with PURL yet)
```

Comment:
This is now available as [`vers:`](https://github.com/package-url/purl-spec/blob/master/VERSION-RANGE-SPEC.rst), but not sure how the two specs play together. Via `&` union?

Reply:
Asked in https://github.com/package-url/purl-spec/issues/66#issuecomment-2630444586